### ISSUE TITLE
Facial model parameters

### ISSUE THREAD

**Issue by GIT-LZL629** [Skype-line/X-Avatar#14]
Can I get the FLAME parameters when building a new avatar?

**Comment by MoyGcc** [comment]
Yes, in principle you can. But it's likely that the parameters are not entirely coherent between the ones of FLAME and the ones of SMPL-X as they're trained separately and might have different spaces. We didn't carefully compare the differences during our development. BTW. SMPL-X has different versions and it's necessary to find the matched SMPL-X and FLAME versions (which have the same dimensions of the facial expressions).
